ARCHITECTURE.md

```diff
@@ -35,7 +38,7 @@ You can find example datasets in [/src/utils/data/](/src/utils/data/), which ill
 
 ## Synchronous data loading
 
-Some data source tokens instruct the app to synchronously import data directly from files in the `src` directory (such as [animals.mock.json](/src/utils/data/animals.mock.json) and [demo.mock.json](/src/utils/data/demo.mock.json)), or generated randomly on page-load. Random data can be seeded with a 'seed' query string in the URL, to allow randomly-generated layouts to be replicated.
+Some data source tokens instruct the app to synchronously import data, which comes direct from files in the `src` directory (such as [animals.mock.json](/src/utils/data/animals.mock.json) and [demo.mock.json](/src/utils/data/demo.mock.json)), or is generated randomly on page-load. Random data can be seeded with a 'seed' query string in the URL, to allow randomly-generated layouts to be replicated.
```

Comment:
> instruct the app to synchronously import data, which comes direct from files in the `src` directory

This is a good stylistic change (avoiding a split infinitive?), but I'm not sure that it improves clarity. There's a few different ways to import data from files and I was trying to imply that I meant [ES6 imports](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import), which is tricky because they don't have a very specific name. I think this is a good reminder that I needed to be more clear here. Perhaps something like this below might work? What do you think?

```suggestion
Some data source tokens instruct the app to synchronously `import` [test](/src/utils/data/animals.mock.json)/[demo](/src/utils/data/demo.mock.json) data from JSON files in the `/src/utils/data` directory, or to generate it randomly on page-load. Random data can be seeded with a 'seed' query string in the URL, to allow randomly-generated layouts to be replicated.
```